documentation/modules/appendix_crds.adoc

```diff
@@ -2024,14 +2024,37 @@ Used in: xref:type-Plugin-{context}[`Plugin`]
 [options="header"]
 |====
 |Property          |Description
-|url        1.2+<.<a|URL of the artifact which will be downloaded. Strimzi does not do any security scanning of the downloaded artifacts. For security reasons, you should first verify the artifacts manually and configure the checksum verification to make sure the same artifact is used in the automated build. Required.
+|url        1.2+<.<a|URL of the artifact which will be downloaded. Strimzi does not do any security scanning of the downloaded artifacts. For security reasons, you should first verify the artifacts manually and configure the checksum verification to make sure the same artifact is used in the automated build. Required. Not applicable for 'maven' type of artifact.
```

Comment:
I don't suppose we can avoid the _Not applicable..._ bit appearing in the sections where it is not relevant?

Reply:
It should appear only in the sections describing jar, zip, tgz and maven type of Connect build.